### PR TITLE
Added My Info | Accept Semina Pini :)

### PR DESCRIPTION
**Pull request by SeminaPXL**
Greetings from Belgium.

### CODE REVIEW

**Comment by whitep4nth3r** [comment]
Hey there! Please could you change your image file extension to `jpeg` as stated in the README? Thank you!

**Comment by whitep4nth3r** [comment]
Also, please don't include your university website — as I would like to keep this to personal websites and promote individuals, and not companies.

**Comment by SeminaPXL** [comment]
@whitep4nth3r 
Done

File: src/_data/profiles/seminapini.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "name": "Semina Pini",
+    "jobTitle": "Student Applied IT Science",
+    "company": "PXL University College",
+    "bio": "Someone who is always looking for new challenges. Someone who believes that JAMStack is the future and is looking for other programmers to work with.",
+    "githubUsername": "SeminaPXL",
+    "twitterUsername": "/",
```
Comment (whitep4nth3r) [inline src/_data/profiles/seminapini.json:7]: If you have no twitter or twitch usernames, please leave black or remove from the file.

Comment (SeminaPXL) [reply src/_data/profiles/seminapini.json]: @whitep4nth3r done

**Pull request merged**